llvm/include/llvm/CodeGen/SelectionDAGISel.h

```diff
@@ -89,9 +89,10 @@ class SelectionDAGISel : public MachineFunctionPass {
   /// not match or is not implemented, return true.  The resultant operands
   /// (which will appear in the machine instruction) should be added to the
   /// OutOps vector.
-  virtual bool SelectInlineAsmMemoryOperand(const SDValue &Op,
-                                            unsigned ConstraintID,
-                                            std::vector<SDValue> &OutOps) {
+  virtual bool
+  SelectInlineAsmMemoryOperand(const SDValue &Op,
+                               const InlineAsm::ConstraintCode ConstraintID,
```

Comment:
Nit: it's weird to mark this parameter itself as const. Not _wrong_ per se, but it's an unusual style, so I think it'd be better to avoid (here and elsewhere).

Reply:
done in https://github.com/llvm/llvm-project/pull/66003/commits/302ca8035512d99a757598af212954495ebb180f